owa/app/js/components/manage_patient_identifier_sources/configureSource.jsx

```diff
@@ -0,0 +1,7 @@
+import React, {Component} from 'react';
+
+export default class ConfigureSource extends Component {
+  render() {
+    return (<p>Configure Content</p>)
+  }
+}
```

Comment:
Please ensure to add a newline to every end of file including this one

Reply:
I have added a new line for each file.